src/index.js

```diff
@@ -1,5 +1,5 @@
 import './style.css';
-import displayUI from './modules/createUI';
+import displayUI from './modules/createUI.js';
 import { getUsers, singleRun } from './modules/displayUser.js';
 import addScore from './modules/addScore.js';
```

Comment:
- [ ] [optional] It would be nice if you could display only the last 5 or 6 scores to make your scoreboard even neater.

Reply:
Thanks